src/ConfigFactory.Core/ConfigModule.cs

```diff
@@ -132,4 +132,15 @@ public virtual bool Validate(out string? message, out ConfigProperty target)
         message = "Validation Successful";
         return true;
     }
+
+    /// <summary>
+    /// Default implementation of the <see cref="IConfigModule.Translate(string)"/>.
+    /// Returns the same same string.
+    /// </summary>
+    /// <param name="input">Input string (key)</param>
+    /// <returns>Translated string</returns>
+    public virtual string Translate(string input)
+    {
+        return input;
```

Comment:
This is fine for the current version, but in future I'd like to work on a default implementation.